fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/VersionsRdfContext.java

```diff
@@ -72,64 +73,41 @@
     public VersionsRdfContext(final FedoraResource resource,
                               final IdentifierConverter<Resource, FedoraResource> idTranslator)
         throws RepositoryException {
-        super();
+        super(idTranslator.reverse().convert(resource).asNode());
         this.idTranslator = idTranslator;
         this.subject = idTranslator.reverse().convert(resource).asNode();
```

Comment:
It is unsettling that `VersionsRdfContext` has its own `private` data member `Node subject` that appears to be eclipsing the data member `Node super.node`.